utils/formatGraph.ts

```diff
@@ -14,19 +14,19 @@ type GraphDataType = {
 
 export default (data: DataType[]) => {
   const graphData: GraphDataType[] = []
-  const today = new Date()
-  const lastMonth = new Date()
-  lastMonth.setMonth(today.getMonth() - 3)
+  const lastMonth = new Date('2020-01-23')
```

Comment:
@kengoy I think this hard code lastMonth date broke the x-axis

Reply:
We're reliving 2020. lol

<img width="706" alt="Screen Shot 2021-01-04 at 9 59 30 AM" src="https://user-images.githubusercontent.com/5452152/103564944-470ae000-4e74-11eb-9a85-b8c80f96ff1b.png">

Reply:
@thuongho Good catch! Glad it is fixed with another PR now.